backoffice/src/Views/Salvor/Edit.cshtml

```diff
@@ -0,0 +1,54 @@
+@model SalvorForm
+@{
+
+    ViewData["Title"] = Model.Id==default(Guid)?"New Salvor":$"Salvor: {Model.Name}";
+}
+
+@{
+    await Html.RenderPartialAsync("_MessageBanner");
+}
+
+<h4>@ViewData["Title"]</h4>
+<form class="form-horizontal mt-4" asp-action="Save" method="POST">
+    @Html.HiddenFor(m => m.Id)
+    <div class="col mt-4">
+        <div class="card">
+            <h5 class="card-header">Salvor</h5>
+            <div class="card-body">
+                <div class="form-group">
+                    @Html.LabelFor(m => m.Name, new { @class = "col-form-label" })
+                    @Html.EditorFor(m => m.Name, new { htmlAttributes = new { @class = "form-control" } })
+                    @Html.ValidationMessageFor(m => m.Name,"", new { @class = "text-danger" })
+                </div>
+                
+                <div class="form-group">
+                    @Html.LabelFor(m => m.Email, new { @class = "col-form-label" })
+                    @Html.EditorFor(m => m.Email, new { htmlAttributes = new { @class = "form-control" } })
+                    @Html.ValidationMessageFor(m => m.Email,"", new { @class = "text-danger" })
+                </div>
+                
+                <div class="form-group">
+                    @Html.LabelFor(m => m.TelephoneNumber, new { @class = "col-form-label" })
+                    @Html.EditorFor(m => m.TelephoneNumber, new { htmlAttributes = new { @class = "form-control" } })
+                    @Html.ValidationMessageFor(m => m.TelephoneNumber,"", new { @class = "text-danger" })
+                </div>
+                
+                <div class="form-group">
+                    @Html.LabelFor(m => m.DateOfBirth, new { @class = "col-form-label" })
+                    @Html.EditorFor(m => m.DateOfBirth, new { htmlAttributes = new { @class = "form-control"} })
+                    @Html.ValidationMessageFor(m => m.DateOfBirth,"", new { @class = "text-danger" })
+                </div>
+                
+                @{
+                    await Html.RenderPartialAsync("_SalvorAddressFormFields",Model);
```

Comment:
Great use of the partial here! 

We should have this as just _AddressFormFields, Model.Address when we have the AddressForm Address on the SalvorForm

Actually it's slightly different, see DroitForm

It'll be like 

<partial name="_AddressFormFields" for="Address"/>

This basically pre-pends the inner object names onto the inputs